<!--
##################################################
Run the object detection system through this launch
file. You can find the parameters below.

(c) Yongyang Nie, 2018
All Rights Reserved.

##################################################
-->

<launch>

    <!--open up the camera -->
	<node name="object_detection_node" pkg="object_detection" type="object_detection_node" output="screen">
        <param name="visualization" value="1" type="bool"/>
        <param name="model_path" value="$(find object_detection)/scripts/model_data/yolov3-tiny-keras.h5"/>
        <param name="anchors_path" value="$(find object_detection)/scripts/model_data/tiny_yolo_anchors.txt"/>
        <param name="classes_path" value="$(find object_detection)/scripts/model_data/coco_classes.txt"/>
        <param name="iou_threshold" value="0.5" type="double"/>
        <param name="score_threshold" value="0.5" type="double"/>
    </node>

</launch>